### DISCUSSION TITLE
💬 Docs Research wants to talk to you!

### DISCUSSION THREAD

**Discussion by supergranular** [community/community#40648] [Projects and Issues]
Hello! Use planning and tracking features? Reference the GitHub issues doc set? We’d like to talk to you.

We’re looking into how people use docs alongside the product. If that’s you and you want to get involved, [start by filling out this 4-question survey](https://github.surveymonkey.com/r/docs-issues-research). We’ll be in touch if you’re a fit.

GitHub Docs Research

**Comment by Lucifer29082002** [comment]
<html>
<body>


  | <!DOCTYPE html>
-- | --
  | <html lang="en">
  | <head>
  | <meta charset="UTF-8">
  | <meta http-equiv="X-UA-Compatible" content="IE=edge">
  | <meta name="viewport" content="width=device-width, initial-scale=1.0">
  | <title>Document</title>
  | <link rel="stylesheet" href="project.css">
  | <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.2.1/css/fontawesome.min.css" integrity="sha384-QYIZto+st3yW+o8+5OHfT6S482Zsvz2WfOzpFSXMF9zqeLcFV0/wlZpMtyFcZALm" crossorigin="anonymous">
  | <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
  | </head>
  | <body style="background-color: rgba(255, 255, 0, 0.179);">
  | <div class="navbar">
  | <img src="C:\Users\91889\Pictures\Untitled design.png" alt="logo" class="logo">
  | <a href="#">Home</a>
  | <a href="#">Carrers</a>
  | <a href="#">About Us</a>
  | <a href="#">Contact</a>
  | <div class="search-bar">
  | <form>
  | <input type="text" placeholder="search..." name="search" class="seaech">
  | <button type="submit"><i class="fa fa-search"></i></button>
  | </form>
  | </div>
  |  
  | </div>
  | <h1 class="achieve"><font size="+22">Achievements</font></h1>
  | <img  class="img11" src="C:\Users\91889\Downloads\arrow_back_ios_FILL0_wght400_GRAD0_opsz48.png" alt="img">
  | <img  class="img22" src="C:\Users\91889\Downloads\arrow_forward_ios_FILL0_wght400_GRAD0_opsz48.png" alt="img2">
  | <img class="img1" src="C:\Users\91889\Pictures\Saved Pictures\img1.jpg" alt="img1">
  | <img class="img2" src="C:\Users\91889\Pictures\Saved Pictures\WhatsApp Image 2022-12-02 at 20.05.11.jpg" alt="img2">
  | <hr>
  | 
  | 
  | 
  | 
  | 
  | <h1 class="h"><font size="+22">Articles</font></h1>
  | <table>
  | <tr>
  | <td><img class="autoResizeImage" src="C:\Users\91889\Pictures\Saved Pictures\akh.jpg" alt="img"></td>
  | <td><img class="autoResizeImage" src="C:\Users\91889\Pictures\Saved Pictures\psp.jpg" alt=""></td>
  | <td><img class="autoResizeImage" src="C:\Users\91889\Pictures\Saved Pictures\WhatsApp Image 2022-12-02 at 20.40.06.jpg" alt=""></td>
  | </tr>
  | </table>
  | <hr>
  |  
  | <footer class="footer">
  | <h4 class="h4">Location and address</h4>
  | <p class="h4"><i>Noida,Delhi <br>India</i></p>
  | <p class="h4"><i>PIN:1123432</i></p>
  |  
  | <h4 class="contact">Contact Details</h4>
  | <p class="contact"><i>Phone no: 8894464829</i></p>
  | <p class="contact"><i>Gmail:faugetshop@gmail.com</i></p>
  |  
  | </footer>
  | </body>
  | </html>


</body>
</html><!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
    <link rel="stylesheet" href="[project.css](https://github.com/community/community/discussions/project.css)">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.2.1/css/fontawesome.min.css" integrity="sha384-QYIZto+st3yW+o8+5OHfT6S482Zsvz2WfOzpFSXMF9zqeLcFV0/wlZpMtyFcZALm" crossorigin="anonymous">
    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
</head>
<body style="background-color: rgba(255, 255, 0, 0.179);">
    <div class="navbar">
        <img src="[C:\Users\91889\Pictures\Untitled design.png](file:///C:/Users/91889/Pictures/Untitled%20design.png)" alt="logo" class="logo">
        <a href="[#](https://github.com/community/community/discussions/40648#)">Home</a>
        <a href="[#](https://github.com/community/community/discussions/40648#)">Carrers</a>
        <a href="[#](https://github.com/community/community/discussions/40648#)">About Us</a>
        <a href="[#](https://github.com/community/community/discussions/40648#)">Contact</a>
        <div class="search-bar">
            <form>
                <input type="text" placeholder="search..." name="search" class="seaech">
                <button type="submit"><i class="fa fa-search"></i></button>
            </form>
    </div>

    </div>
    <h1 class="achieve"><font size="+22">Achievements</font></h1>
     <img  class="img11" src="[C:\Users\91889\Downloads\arrow_back_ios_FILL0_wght400_GRAD0_opsz48.png](file:///C:/Users/91889/Downloads/arrow_back_ios_FILL0_wght400_GRAD0_opsz48.png)" alt="img">
    <img  class="img22" src="[C:\Users\91889\Downloads\arrow_forward_ios_FILL0_wght400_GRAD0_opsz48.png](file:///C:/Users/91889/Downloads/arrow_forward_ios_FILL0_wght400_GRAD0_opsz48.png)" alt="img2">
    <img class="img1" src="[C:\Users\91889\Pictures\Saved Pictures\img1.jpg](file:///C:/Users/91889/Pictures/Saved%20Pictures/img1.jpg)" alt="img1">
    <img class="img2" src="[C:\Users\91889\Pictures\Saved Pictures\WhatsApp Image 2022-12-02 at 20.05.11.jpg](file:///C:/Users/91889/Pictures/Saved%20Pictures/WhatsApp%20Image%202022-12-02%20at%2020.05.11.jpg)" alt="img2">
    <hr>
    
        
        
        
    
    <h1 class="h"><font size="+22">Articles</font></h1>
    <table>
        <tr>
            <td><img class="autoResizeImage" src="[C:\Users\91889\Pictures\Saved Pictures\akh.jpg](file:///C:/Users/91889/Pictures/Saved%20Pictures/akh.jpg)" alt="img"></td>
            <td><img class="autoResizeImage" src="[C:\Users\91889\Pictures\Saved Pictures\psp.jpg](file:///C:/Users/91889/Pictures/Saved%20Pictures/psp.jpg)" alt=""></td>
            <td><img class="autoResizeImage" src="[C:\Users\91889\Pictures\Saved Pictures\WhatsApp Image 2022-12-02 at 20.40.06.jpg](file:///C:/Users/91889/Pictures/Saved%20Pictures/WhatsApp%20Image%202022-12-02%20at%2020.40.06.jpg)" alt=""></td>
        </tr>
    </table>
    <hr>

    <footer class="footer">
        <h4 class="h4">Location and address</h4>
        <p class="h4"><i>Noida,Delhi <br>India</i></p>
        <p class="h4"><i>PIN:1123432</i></p>

        <h4 class="contact">Contact Details</h4>
        <p class="contact"><i>Phone no: 8894464829</i></p>
        <p class="contact"><i>Gmail:faugetshop@gmail.com</i></p>

    </footer>
</body>
</html>

**Reply by Bray40** [reply]
Brayanafernandez8@gmail.com
